vector/src/main/java/im/vector/riotredesign/features/roomdirectory/PublicRoomItem.kt

```diff
@@ -63,6 +63,8 @@ abstract class PublicRoomItem : VectorEpoxyModel<PublicRoomItem.Holder>() {
 
         AvatarRenderer.render(avatarUrl, roomId!!, roomName, holder.avatarView)
```

Comment:
Might be needed to clear existing glide requests on this view? (or do it on unbinding?)

Reply:
Maybe open a specific issue to track this, out of scope of this PR